front/src/applications/operationalStudies/views/SimulationResults.tsx

```diff
@@ -201,12 +203,7 @@ const SimulationResults = ({
                     <ManchetteWithSpaceTimeChartWrapper
                       operationalPoints={projectedOperationalPoints}
                       projectPathTrainResult={projectPathTrainResult}
-                      // TODO Paced train : remove this condition in https://github.com/OpenRailAssociation/osrd/issues/10613
-                      selectedTrainScheduleId={
-                        selectedTimetableItem && isTrainSchedule(selectedTimetableItem.id)
-                          ? selectedTimetableItem.id
-                          : undefined
-                      }
+                      selectedTrainId={selectedTrainId}
```

Comment:
I think it would be better to use `selectedTimetableItem`, since it should be equal to `selectedTrainId` and we already have `selectedTimetableItem` in the component
```suggestion
                      selectedTrainId={selectedTimetableItem.id}
```

Reply:
I think not, because it's not the same type at all. 
`selectedTimetableItem` is of type `TimetableItemWithTimetableId` who can be related to `PacedTrain` or `TrainSchedule` but here we want a `TrainId` who is related to `Occurrence` and `TrainSchedule`